pdns/nod.cc

```diff
@@ -34,12 +34,23 @@
 #include "misc.hh"
 
 using namespace nod;
-using namespace boost::filesystem;
+namespace filesystem = boost::filesystem;
 
 // PersistentSBF Implementation 
 
 std::mutex PersistentSBF::d_cachedir_mutex;
 
+void PersistentSBF::remove_tmp_files()
```

Comment:
Might be nice to add a comment stating that we need to hold the mutex while calling that function?

Reply:
Or take a `std::lock_guard<std::mutex>&` parameter to actually enforce that? And perhaps a `const filesystem::path&` while we are at it?